app/scripts/migrations/index.js

```diff
@@ -72,6 +72,7 @@ import m065 from './065';
 import m066 from './066';
 import m067 from './067';
 import m068 from './068';
+import m069 from './069';
 
 const migrations = [
```

Comment:
Is it worth replacing this with a dynamic import at some point?

Reply:
Hm, what do you have in mind? Like, dynamically importing all migrations? That's an improvement I could get behind.

Reply:
Yeah, looping through all of the files inside of this folder and then dynamically importing them instead of listing them by hand. Not proposing this in the current PR, mind, just an idea as I read this.